tests/nvme_test.py

```diff
@@ -309,12 +309,12 @@ def test_connect_single_ns(self):
         # nothing to connect to
         with self.assertRaisesRegexp(GLib.GError, r"Error connecting the controller: "):
             # FIXME: libnvme itself spits an error on stdout
-            BlockDev.nvme_connect(self.SUBNQN, 'loop', None, None, None, None, self.HOSTNQN, None)
+            BlockDev.nvme_connect(self.SUBNQN, 'loop', None, None, None, None, self.HOSTNQN, None, None)
```

Comment:
Also having a test that actually checks the extra arguments work would be nice.

Reply:
Yeah, I knew about that, but with the `loop` transport most of the attributes get ignored. I think I haven't found anything that really worked. I'll keep it in mind, I have some more planned changes that might actually use the extra arguments (ssl, auth?)